addons/HotelPvP/scripts/Game/AFM_HotelPvP/Editor/Components/Editor/AFM_CameraLimitedEditorComponent.c

```diff
@@ -0,0 +1,84 @@
+modded class SCR_CameraLimitedEditorComponent : SCR_CameraEditorComponent
```

Comment:
Rename the file to `SCR_CameraLimitedEditorComponent`, it does not hold a new AFM class but mods vanilla class so the naming should reflect that.